packages/manager/modules/pci/src/projects/project/storages/databases/database/backups/fork/command/command.controller.js

```diff
@@ -0,0 +1,35 @@
+import {
+  API_GUIDES,
+  DATABASE_CREATION_GUIDES,
+} from '../../../../../../project.constants';
+
+import { getOrderDataFromModel } from '../../../../add/add.utils';
+import { ORDER_KEYS } from '../fork.constants';
+
+export default class ForkCommandCtrl {
+  /* @ngInject  */
+  constructor($translate, DatabaseService) {
+    this.$translate = $translate;
+    this.orderKeys = ORDER_KEYS;
+    this.DatabaseService = DatabaseService;
+  }
+
```

Comment:
```suggestion
  constructor() {
    this.orderKeys = ORDER_KEYS;
  }

```
I think doesn't used.

Reply:
Hello, totally right, fix :)